src/auth/refreshclient.ts

```diff
@@ -21,7 +21,9 @@ import {BodyResponseCallback} from './../transporters';
 import {OAuth2Client} from './oauth2client';
 
 export class UserRefreshClient extends OAuth2Client {
-  public _refreshToken: string;
+  // TODO: refactor tests to make this private
+  // tslint:disable-next-line
```

Comment:
let be specific about the lint rule you want to disable. BTW, note that gts 0.5.0 will no longer complain about this.

Reply:
fixed!